diesel/Cargo.toml

```diff
@@ -17,7 +17,7 @@ byteorder = { version = "1.0", optional = true }
 chrono = { version = "0.4.20", optional = true, default-features = false, features = ["clock", "std"] }
 libc = { version = "0.2.0", optional = true }
 libsqlite3-sys = { version = ">=0.17.2, <0.29.0", optional = true, features = ["bundled_bindings"] }
-mysqlclient-sys = { version = "0.2.5", optional = true }
+mysqlclient-sys = { git = "https://github.com/xiangzhai/mysqlclient-sys.git", branch = "rust-bindgen", optional = true }
```

Comment:
It's not possible to depend on a forked git version here. These patches need to be included into the mysqlclient-sys repo.

(Open a PR there and we can discuss the changes there)

Reply:
RFR:  https://github.com/sgrif/mysqlclient-sys/pull/37